app/models/user.rb

```diff
@@ -7,4 +7,11 @@ class User < ApplicationRecord
   validates :password, presence: true, length: { minimum: 6 }
   before_save { self.email = email.downcase }
   has_secure_password
+
+  def User.digest(string) # Returns the hash digest of the given string.
+    cost = ActiveModel::SecurePassword.min_cost ?
```

Comment:
if you using a ternary operator pls write of a single line.
if the line turned out too long then use block operator `if-else`